requests/src/test/scala/uk/ac/wellcome/platform/api/requests/RequestingScenarioTest.scala

```diff
@@ -296,6 +296,269 @@ class RequestingScenarioTest
+             |}
+             |""".stripMargin
+        )
+      }
+    }
+
+    Scenario("An item that has been suppressed in Sierra") {
+      // Like the previous scenario, this should be unlikely in practice, but we include
+      // the test for completeness.
+
+      Given("a physical item from Sierra in the catalogue API")
+      val patronNumber = createSierraPatronNumber
+      val itemNumber = createSierraItemNumber
+      val item = createIdentifiedSierraItemWith(itemNumber)
+      val lookup = new MemoryItemLookup(items = Seq(item))
+
+      And("which is suppressed in Sierra")
+      val responses = Seq(
+        (
+          createHoldRequest(patronNumber, itemNumber),
+          HttpResponse(
+            status = StatusCodes.InternalServerError,
+            entity = HttpEntity(
+              contentType = ContentTypes.`application/json`,
+              """
+                |{
+                |  "code": 132,
+                |  "specificCode": 2,
+                |  "httpStatus": 500,
+                |  "name": "XCirc error",
+                |  "description": "XCirc error : This record is not available"
+                |}
+                |""".stripMargin
+            )
+          )
+        ),
+        (
+          createListHoldsRequest(patronNumber),
+          createListHoldsResponse(patronNumber, items = Seq())
+        ),
+        (
+          HttpRequest(uri =
+            s"http://sierra:1234/v5/items/$itemNumber?fields=deleted,status,suppressed"),
+          HttpResponse(
+            entity = HttpEntity(
+              contentType = ContentTypes.`application/json`,
+              s"""
+                 |{
+                 |  "id": "$itemNumber",
+                 |  "deletedDate": "2001-01-01",
+                 |  "deleted": false,
+                 |  "suppressed": true,
+                 |  "status": {"code": "-", "display": "Available"}
+                 |}
+                 |""".stripMargin
+            )
+          )
+        )
+      )
+
+      implicit val route: Route = createRoute(lookup, responses)
+
+      When("the user requests the item")
+      val response = makePostRequest(
+        path = s"/users/$patronNumber/item-requests",
+        entity = createJsonHttpEntityWith(
+          s"""
+             |{
+             |  "itemId": "${item.id.canonicalId}",
+             |  "workId": "$createCanonicalId",
+             |  "type": "ItemRequest"
+             |}
+             |""".stripMargin
+        )
+      )
+
+      Then("the hold is rejected")
+      response.status shouldBe StatusCodes.BadRequest
+      response.status.intValue shouldBe 400
+
+      And("the error explains why the hold is rejected")
+      withStringEntity(response.entity) {
+        assertJsonStringsAreEqual(
+          _,
+          s"""
+             |{
+             |  "@context": "$contextUrl",
+             |  "type": "Error",
+             |  "errorType": "http",
+             |  "httpStatus": 400,
```

Comment:
Is there an argument that this could be a 409?

Reply:
I think the plan is to use 409 Conflict for cases where the item is on hold to somebody else.